flink-runtime-web/web-dashboard/src/app/components/humanize-watermark.pipe.ts

```diff
@@ -35,3 +35,19 @@ export class HumanizeWatermarkPipe implements PipeTransform {
     }
   }
 }
+
+@Pipe({
+  name: 'humanizeWatermarkToDate',
+  standalone: true
+})
+export class HumanizeWatermarkToDatePipe implements PipeTransform {
```

Comment:
Same here, maybe `HumanizeWatermarkToDatetimePipe` is better.

Reply:
OK, that makes sense.